css/style.css

```diff
@@ -1,4 +1,384 @@
+  padding-right: 40px;
+}
+
+.data__road-block {
+padding-bottom: 4px;
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+  padding-right: 3px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 21px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 13px;
+}
+
+.test-btn {
+  display: flex;
+  padding-right: 19px;
+}
+
+.testdrive {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  white-space: nowrap;
+  width: 118px;
+  margin-right: 40px;
+}
+
+.testdrive__img {
+  width: 22px;
+  height: 22px;
+  padding-right: 6px;
+}
+
+.testdrive__text {
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.order-btn {
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  transition: all 0.5s ease 0s; 
+}
+
+.order-btn:hover {
+  cursor: pointer;
+  border: none;
+  color: #fff;
+  background-color: #e51937;
+  transition: all 0.5s ease 0s; 
+  box-shadow: 0px 3px 3px 1px #9b9999;
+}
+
+.arrow {
+  background-color: #fff;
+  width: 24px;
+  height: 15px;
+  position: relative;
+  transition: all 0.5s ease 0s;
+  margin: auto;
+}
+
+.arrow__line {
+  position: absolute;
+  width: 22px;
+  height: 3px;
+  background-color: #e51937;
+}
+
+.first-ln {
+  transform: translate(-7px, 7px) rotate(45deg);
+  top: 7px; 
+}
+
+.second-ln {
+  transform: translate(7px, -7px) rotate(-45deg); 
+  top: 21px;
+}
+
+.arrow:hover {
+  cursor: pointer;
+}
+
+
+@media (min-width:1200px) and (max-width:1295px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 170px;
+  }
+  .tesla-bgd{
+    height: auto;
+    width: 100%;
+    max-width: 1295px;
+    top: -103px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 23px 0 20px 107px;
+  }
+  .content__data-car {
+    padding: 0 232px 22px 199px;
+  }
+
+}
+@media (min-width:1024px) and (max-width:1199px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 120px;
+  }
+  .tesla-bgd, .tesla-car {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+  .tesla-bgd{
+    top: -75px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 65px 0 20px 4px;
+  }
+  .content__data-car {
+    padding: 0 81px 22px 0px;
+  }
+  .content__facts {
+    padding: 3px 0 0 156px;
+  }
+  .data__speed-block {
+    padding-right: 88px;
+  }
+  .data__specifications {
+    display: block;
+  }
+  .test-btn {
+    padding-right: 42px;
+  }
+  .testdrive {
+    margin-right: 60px;
+  }
+}
+@media (min-width:900px) and (max-width:1023px) {
+
+}
+@media (min-width:768px) and (max-width:899px) {
+  .header {
+    padding: 25px 24px 96px 0;
+  }
+  .header__logo-img{
+    padding-left: 15px;
+    width: 167px;
+    height: 50px;
+  }
+  .menubar__link {
+    display: none;
+  }
+  .button {
+    width: 33px;
+    height: 20px;
+  }
+  .second {
+    top: 10px;
+  }
+  .third {
+    top: 19px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)